specs/CMakeLists.txt

```diff
@@ -169,6 +169,7 @@ function(generateNativeTables)
     "linux/elf_sections.table:linux"
     "linux/selinux_settings.table:linux"
     "linux/apparmor_profiles.table:linux"
+    "linux/startup_items.table:linux"
```

Comment:
Remove this line, and add `linux` to line 177, also move `./specs/macwin/startup_items.table` to `./specs`